metaflow/plugins/aip/aip_metaflow_step.py

```diff
@@ -398,6 +427,28 @@ def aip_metaflow_step(
         with open(output_file, "w") as f:
             f.write(str(values[idx]))
 
+    # get card and write to output file
+    card_cli: str = _get_cards_cli(
+        step_name,
+        task_id,
+        metaflow_run_id,
+        script_name,
+    )
+    cmd: str = card_cli.format(run_id=metaflow_run_id)
+    cmd = f"{cmd} > /tmp/outputs/cards/default_card.html"
+
+    pathlib.Path("/tmp/outputs/cards/").mkdir(parents=True, exist_ok=True)
+    with Popen(
+        cmd, shell=True, universal_newlines=True, executable="/bin/bash", env=env
+    ) as process:
+        pass
+
+    if process.returncode != 0:
+        logging.info(f"---- Following command returned: {process.returncode}")
+        logging.info(cmd.replace(" && ", "\n"))
```

Comment:
Maybe `cmd.replace(" && ", "\n&& ")` so that you get new lines without losing the command chaining semantics.

Reply:
this is a nice to have, but skipped it for this change, as it already exists and is for debugging only

thank you!